From bcec4cc957874cfc0e0507510a5cc541c364f1f0 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:49:21 -0400
Subject: [tools] perf/kvm: Fix spurious '=' use in man page

Message-id: <1408953210-26343-74-git-send-email-jolsa@redhat.com>
Patchwork-id: 88802
O-Subject: [PATCH RHEL7.1 BZ1133083 073/322] perf kvm: Fix spurious '=' use in man page
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit 316bd98a9aa47693215e0d3a5bcc83f51691ba93
Author: Dongsheng Yang <yangds.fnst@cn.fujitsu.com>
Date: Wed Dec 4 17:56:43 2013 -0500

description
===========
As option --host and --guest request no input for it, there should not
be a '=' after them in the man page sources.

And --output expects a filename as the input, so there should be a '='
after it.

This patch removes the needless '=' after --guest and --host, and adds a
'=' after --output in perf-kvm.txt.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/Documentation/perf-kvm.txt b/tools/perf/Documentation/perf-kvm.txt
index f52f94d..7953227 100644
--- a/tools/perf/Documentation/perf-kvm.txt
+++ b/tools/perf/Documentation/perf-kvm.txt
@@ -60,14 +60,14 @@ There are a couple of variants of perf kvm:
 OPTIONS
 -------
 -i::
---input=::
+--input=<path>::
         Input file name.
 -o::
---output::
+--output=<path>::
         Output file name.
---host=::
+--host::
         Collect host side performance profile.
---guest=::
+--guest::
         Collect guest side performance profile.
 --guestmount=<path>::
  Guest os root file system mount directory. Users mounts guest os
-- 
1.7.1